src/ceph_mon.cc

```diff
@@ -356,6 +356,8 @@ int main(int argc, const char **argv)
     exit(1);
   }
 
+  auto store = std::make_shared<MonitorDBStore>(g_conf()->mon_data);
```

Comment:
make_unique?

Reply:
I don't see a reason not to, same for the Throttles. Updated.